### PR TITLE
[WIP] Solicitor service method

### PR DESCRIPTION
**Pull request by mounikahmcts**
-  https://tools.hmcts.net/jira/browse/DIV-4961
-  https://tools.hmcts.net/jira/browse/DIV-5268

### CODE REVIEW

File: definitions/divorce/json/CaseEventToFields.json [hunk 1]
```diff
@@ -7276,7 +7313,9 @@
     "PageLabel": "Grant Decree Absolute",
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
-    "FieldShowCondition": "DecreeAbsoluteGranted=\"nevershow\""
+    "FieldShowCondition": "DecreeAbsoluteGranted=\"nevershow\"",
+    "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/sol-dn-review-petition",
+    "RetriesTimeoutURLMidEvent": 120120
```
Comment (qzhou-hmcts) [inline definitions/divorce/json/CaseEventToFields.json]: "120,120"

Comment (mounikahmcts) [reply definitions/divorce/json/CaseEventToFields.json]: Existing files had 120120 and updated all refernces to "120,120" now

File: definitions/divorce/json/AuthorisationCaseField.json [hunk 2]
```diff
@@ -12423,5 +12423,131 @@
...
+    "LiveFrom": "19/07/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "LabelSolPersonalService",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "19/07/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "LabelSolPersonalService",
```
Comment (alexwakeman) [inline definitions/divorce/json/AuthorisationCaseField.json]: Does this label need to be CRUD access?

Comment (mounikahmcts) [reply definitions/divorce/json/AuthorisationCaseField.json]: > Does this label need to be CRUD access?

Labels dont need to have delete access so removed them

Comment (alexwakeman) [reply definitions/divorce/json/AuthorisationCaseField.json]: Actually it should be just R - as we can't update or create labels in CCD, only in the config file right?

Comment (mounikahmcts) [reply definitions/divorce/json/AuthorisationCaseField.json]: > Actually it should be just R - as we can't update or create labels in CCD, only in the config file right?

Though i did not find any difference in the cases when it is set only to R access, as of now  i see all labels are having CRU access.

File: definitions/divorce/json/AuthorisationCaseField.json [hunk 3]
```diff
@@ -12423,5 +12423,131 @@
...
     "UserRole": "caseworker-divorce-courtadmin-la",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "19/07/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "LabelSolServiceMethod",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "R"
+  },
```
Comment (hosainnet) [inline definitions/divorce/json/AuthorisationCaseField.json]: Do we know if this is going live in the next release (around the ~11th of Aug) or sometime later? If it's not going live soon I would personally suggest we keep it in the branch - cherry-pick to prod last time was a little tricky and took time :)

File: definitions/divorce/json/AuthorisationCaseField.json [hunk 4]
```diff
@@ -12423,5 +12423,131 @@
...
+    "CaseFieldID": "LabelSolPersonalService",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "19/07/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "LabelSolPersonalService",
+    "UserRole": "caseworker-divorce-solicitor",
+    "CRUD": "CRU"
```
Comment (hosainnet) [inline definitions/divorce/json/AuthorisationCaseField.json]: Just need R here right?

File: definitions/divorce/json/AuthorisationCaseField.json (hunk 2 above)
Comment (alexwakeman) [reply definitions/divorce/json/AuthorisationCaseField.json]: Fair enough, I think this is a mistake as labels are Readonly and static.

File: package.json [hunk 5]
```diff
@@ -33,7 +33,7 @@
     }
   },
   "scripts": {
-    "setup": "git submodule update --init --recursive && git submodule foreach --recursive git pull origin master && yarn --cwd ccd-definition-processor install",
+    "setup": "git submodule update --init --recursive && yarn --cwd ccd-definition-processor install",
```
Comment (hosainnet) [inline package.json]: This should be in master so worth updating the branch 👍

File: definitions/divorce/json/AuthorisationCaseField.json (hunk 2 above)
Comment (mounikahmcts) [reply definitions/divorce/json/AuthorisationCaseField.json]: > Fair enough, I think this is a mistake as labels are Readonly and static.

Moved labels in this PR to read only access

File: definitions/divorce/json/AuthorisationCaseField.json (hunk 3 above)
Comment (mounikahmcts) [reply definitions/divorce/json/AuthorisationCaseField.json]: These changes will not go as part of "Respondent Solicitor release" (11th august).

Should release these changes as part of "Solicitor - Personal Service"

**Pull request merged**